website/cypress/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["es5", "dom"],
+    "types": ["cypress", "node"],
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "jsx": "react-jsx"
```

Comment:
The regular `tsconfig.json` has `baseUrl: "."` could that work here too?

Reply:
Adding `baseUrl: ".."` worked to avoid relative imports for the contract and the component test.